### PR TITLE
feat(general-petitions): General petition ---> minar sidur

### PR DESCRIPTION
**Pull request by albinagu**
## What

General Petition on Mínar Síður

PS. Also updating a couple of queries for the general petition system and removing mocks

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by kksteini — APPROVED** [review]
LGTM

File: libs/application/templates/general-petition/src/lib/generalPetitionApplicationTemplate.ts [hunk 1]
```diff
@@ -73,6 +73,14 @@ const GeneralPetitionApplicationTemplate: ApplicationTemplate<
                 ),
               read: 'all',
             },
+            {
+              id: Roles.SIGNATUREE,
+              formLoader: () =>
+                import('../forms/EndorsementForm').then((val) =>
+                  Promise.resolve(val.EndorsementForm),
+                ),
+              read: 'all',
```
Comment (eirikurn) [inline libs/application/templates/general-petition/src/lib/generalPetitionApplicationTemplate.ts:82]: Is there anything sensitive here which we don't want the signaturee to have access to?

File: libs/service-portal/endorsements/src/screens/ViewPetition/ViewPetition.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                <Button
+                  icon="lockClosed"
+                  iconType="outline"
+                  colorScheme="destructive"
+                >
+                  {formatMessage(m.viewPetition.closeListButton)}
+                </Button>
+              }
+              onConfirm={() => console.log('Confirmed')}
+              onCancel={() => console.log('Cancelled')}
```
[the remark this answers is not in the document]
Comment (albinagu) [reply libs/service-portal/endorsements/src/screens/ViewPetition/ViewPetition.tsx:182]: yes, feature flag is the plan for now

**Review by oddsson — APPROVED** [review]

**Review by thordurhhh — APPROVED** [review]

File: libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx [hunk 3]
```diff
@@ -87,7 +87,14 @@ export const ActionCard: React.FC<ActionCardProps> = ({
   const progressMeter = { ...defaultProgressMeter, ..._progressMeter }
   const tag = { ...defaultTag, ..._tag }
   const unavailable = { ...defaultUnavailable, ..._unavailable }
-  const bgr = backgroundColor === 'white' ? 'white' : 'blue100'
+  const bgr =
+    backgroundColor === 'white'
+      ? 'white'
+      : backgroundColor === 'blue'
+      ? 'blue100'
```
Comment (maranomynet) [inline libs/island-ui/core/src/lib/ActionCard/ActionCard.tsx:94]:
```suggestion
```
Already covered by the default case.

**Review by maranomynet — APPROVED** [review]

**Pull request merged**